### PR TITLE
Add openLR option to normalize-ways script

### PR DESCRIPTION
**Pull request by nickcordella**
this adds the ability add openlr strings to version 2.3.4

### CODE REVIEW

File: bin/normalize-ways [hunk 1]
```diff
@@ -74,10 +76,16 @@ waysStream.on('data', function (line) {
...
       way.properties.id = way.properties.id + ';' + quadkey;
+      if (openLR) {
+        way.properties.openLR = geojsonLR.toOpenLRString(way);
+      }
       outputStream.write(JSON.stringify(way) + '\n');
     }
     way = ways[ways.length - 1];
     way.properties.id = way.properties.id + ';' + quadkey;
+    if (openLR) {
+      way.properties.openLR = geojsonLR.toOpenLRString(way);
```
Comment (camilosanin) [inline bin/normalize-ways]: do you think there is a way to avoid having to do this line (and the one before) twice?

Comment (camilosanin) [reply bin/normalize-ways]: Lines 84-90 are a strange way of adding the last line.  I know it was not part of this PR, but it'd be awesome if there is a way of avoiding it. All good if you think this is OK for now.

Comment (nickcordella) [reply bin/normalize-ways]: yes I agree there is probably a more elegant pattern here but let's save it for a different PR

File: bin/normalize-ways [hunk 2]
```diff
@@ -5,6 +5,7 @@
 var byline = require('byline');
 var fs = require('graceful-fs');
 var path = require('path');
+var geojsonLR = require('@mapbox/geojson-lr');
```
Comment (benjamintd) [inline bin/normalize-ways]: @nickcordella you're adding a closed source dependency to an open source library ;)

**Pull request merged**